docs/aws/README.md

```diff
@@ -106,8 +113,13 @@ EOF
 kubectl get provisioner default -o yaml
 ```
 
-### Create some pods
-Create some dummy pods and observe logs.
+# First Use
+
+Karpenter is now active and ready to begin provisioning nodes. Create a workload (e.g., deployment) to see karpenter provision some nodes. 
+
+## Create Workload
+
+This deployment is based on the pause image and initially has zero replicas. 
```

Comment:
Consider: 
"This deployment uses the [pause image](https://www.ianlewis.org/en/almighty-pause-container) and starts with zero replicas."